data/ke2022daily/ke2022daily.yaml

```diff
@@ -0,0 +1,5379 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Daily Longitudinal Sampling of SARS-CoV-2 Infection Reveals Substantial Heterogeneity
+  in Infectiousness
+doi: 10.1038/s41564-022-01105-z
+description: >
+  The authors studied the dynamics of infectious virus and viral RNA shedding for
+  SARS-CoV-2 during acute infection through daily longitudinal sampling of 60 individuals
+  for up to 14 days. Nasal swab and saliva samples were collected daily and tested
+  for University of Illinois at Urbana-Champaign faculty, staff, and students (during
+  the fall of 2020 and spring of 2021) who reported a negative RT-qPCR test result
+  in the past 7 days and were either within 24 h of a positive RT-qPCR result or within
+  5 days of exposure to someone with a confirmed positive RT-qPCR result.
+analytes:
+  nasal_SARSCoV2:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration in mid-turbinate nasal swab (nasopharyngeal
+      swab) samples. Note that the unit of these measurements is per mL: this is because
+      nasal swab samples were each collected in 3 mL of VTM. The calibration curve
+      for nasal samples was in the Supplemental Table S10 in Ke et al. (2022).
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N1 and N2
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    unit: gc/mL
+    reference_event: confirmation date
```

Comment:
Very cool to have data for the earlier infection course. Do they also report the day of symptom onset for each patient? It would be interesting to be able to convert between the two different reference events when we analyze data together with other studies.

Reply:
Good point! They did have a figure of individual-level symptom data (https://www.nature.com/articles/s41564-022-01105-z/figures/6). Early day symptom data could be missing for some patients. Should I take the first day of any symptom for each patient?

Reply:
That sounds like a good plan! In terms of adding the data, how do you think we should add the data? Maybe something like a patient-level attribute that encodes the number of days between confirmation date and symptom onset?

Reply:
I think that's a reasonable solution. I can also try to use the `symptom onset` as the `reference_event`. But it may not exist for all the patients.

Reply:
I modified all the `time` using the `reference_event` as `symptom_onset` and added a new field for each measurement to indicate the confirmation gap after onset date.